src/main/java/build/buildfarm/common/redis/RedisQueue.java

```diff
@@ -54,6 +54,15 @@ public void push(JedisCluster jedis, String val) {
     jedis.lpush(name, val);
   }
 
+  /**
+   * @brief Push a value onto the queue.
+   * @details Adds the value into the backend redis queue.
+   * @param val The value to push onto the queue.
+   */
+  public void push(JedisCluster jedis, String val, double priority) {
+    throw new IllegalArgumentException("That's not a priority queue");
```

Comment:
actually, can you do a regular push here instead.  Simply ignore the priority.
We'll be forwarding the priority queue data in later.  It will be easier for us to migrate data structures when the client is already calling this API.